src/frequenz/client/base/client.py

```diff
@@ -222,12 +247,37 @@ def connect(self, server_url: str | None = None) -> None:
         Args:
             server_url: The URL of the server to connect to. If not provided, the
                 previously used URL is used.
+            auth_key: The API key to use when connecting to the service.
+            sign_secret: The secret to use when creating message HMAC.
```

Comment:
I would clarify in here that the old ones are kept if they are not passed or `...` is passed. While at it, maybe it is worth to homogenize this and change `server_url: str | EllipsisType = ...`, and then we can just mention once in the docs body that any omitted value (or value passed as `...`) will keep the old value. But we can change the `server_url` in a separate PR too.